live-hls.js

```diff
@@ -27,7 +27,8 @@ var defaults = {
   tsnotfound: 0,
   manifestnotfound: 0,
   resetStream: 0,
-  stopStream: 0
+  stopStream: 0,
```

Comment:
A general set of thoughts after reading through this file since a "clean up Murphy" story was suggested.

We could pretty easily jump this into the land of Node 8 to get the benefits of ES6.  Could dump these `vars`, use destructuring in parts, arrow functions, default values on functions, etc.  That would probably help cut a bit of code out.

I also think it might be nice to add some `.eslint` checking to this project to ensure everything is consistent.

Reply:
I'll keep this PR saved for reference for when we update to ES6. These are good suggestions.